api/framework/statsgrid2016/bundle.md

```diff
@@ -28,13 +28,18 @@ Indicator selector, regionset selector and a datagrid with the indicator data ar
           "type": "system"
         }],
         "grid": true,
-        allowClassification : true
+        "allowClassification" : true,
+        "vectorViewer": true,
+        "scrollDatatable": true
```

Comment:
Need to think the config a bit, I think the scrollDatatable isn't very good configurable thing. Also vectorViewer could be tied to the map NOT having the StatsLayerPlugin registered OR RegionsetViewer.js could be an alternative implementation of StatsLayerPlugin.